go-controller/pkg/clustermanager/userdefinednetwork/template/net-attach-def-template_test.go

```diff
@@ -27,6 +34,7 @@ var _ = Describe("NetAttachDefTemplate", func() {
 					Subnets: udnv1.DualStackCIDRs{"abc"},
 				},
 			},
+			"CIDR \"abc\" not properly formatted",
```

Comment:
IMO: such kind of test cases can be removed if they are covered by  CRD validations.